### PR TITLE
Fix compie failed when you define macro QT_NAMESPACE

### PR DESCRIPTION
**Pull request by zccrs**
My Qt library define macro QT_NAMESPACE, This led to compile failed

### CODE REVIEW

**Comment by pvuorela** [comment]
Includes build directory.

File: connection/dbuscustomarguments.h [hunk 1]
```diff
@@ -14,14 +14,20 @@
...
+
 class MImPluginSettingsEntry;
 class MImPluginSettingsInfo;
+
+QT_BEGIN_NAMESPACE
 class QDBusArgument;
 class QVariant;
+QT_END_NAMESPACE
 
 #include <maliit/namespace.h>
```
Comment (pvuorela) [inline connection/dbuscustomarguments.h]: Could move this up so no two separate qt namespace blocks would be needed.

**Comment by zccrs** [comment]
Thank you, I have re push code

**Comment by pvuorela** [comment]
Looks good to me now.

**Pull request merged**